code/game/jobs/job/medical.dm

```diff
@@ -130,13 +129,13 @@
 
 	total_positions = 1
 	spawn_positions = 1
-	economic_modifier = 5
+
 	minimal_player_age = 3
 	supervisors = "the chief medical officer"
 	selection_color = "#013d3b"
 	access = list(access_medical, access_medical_equip, access_morgue, access_surgery, access_chemistry, access_virology, access_genetics, access_psychiatrist)
 	minimal_access = list(access_medical, access_medical_equip, access_psychiatrist)
-	alt_titles = list("Psychologist" = /decl/hierarchy/outfit/job/medical/psychiatrist/psychologist)
+
```

Comment:
A psychosis system is apparently planned, and thus there may be mechanical purpose for psychology yet. Best to get input on this

Reply:
I think we're wrapping that duty into the Medical Doctor role rather than specialised psychologists. Runs the risk of Baycode's iconic feature bloat otherwise. But yeah, if West wants to give input.

Reply:
Psychology duties and the relative tasks that will be a result of the psychosis system (to be added) will be performed by Medical Doctors and Researchers.